lib/pay/stripe/webhooks/payment_failed.rb

```diff
@@ -0,0 +1,23 @@
+module Pay
+  module Stripe
+    module Webhooks
+      class PaymentFailed
+        def call(event)
+          # Event is of type "invoice" see:
+          # https://stripe.com/docs/api/invoices/object
+
+          object = event.data.object
```

Comment:
One thing we need to consider here:

The `attempt_count` field on the invoice shows the number of attempts to pay this invoice. I read somewhere in the docs that the first one can be from the initial attempt (like if someone is subscribing and their first attempt fails maybe?). That might need to skip the first attempt or something, but wanted to make a note about it.

That will also be useful for changing the text for these templates. Maybe the `payment_failed` mailer takes the attempt count and determines which template to render? Or we have separate `payment_failed_first`, `payment_failed_second`, etc mailers?